README.md

```diff
@@ -14,7 +16,7 @@ The easiest way to get started with Nex is to check out our [Using Nex](https://
 If you're already familiar with Nex and how it works, then you can get going quickly by installing the Nex CLI:
 
 ```
-go install github.com/synadia-io/nex/nex@latest
+curl -sSf https://nex.synadia.com/install.sh | sh 
 ```
 
 _**Note** that an installation method that doesn't require Go is coming soon._
```

Comment:
we should probably remove this note since we're adding the non-Go installation method :)